### ISSUE TITLE
Uncaught TypeError: Cannot read property 'msie' of undefined

### ISSUE THREAD

**Issue by charl** [koenpunt/chosen#26]
With jQuery v1.9 or greater you get the following error:

```
Uncaught TypeError: Cannot read property 'msie' of undefined
```

The line it points to is:

``` js
if ($.browser.msie && ($.browser.version === "6.0" || ($.browser.version === "7.0" && document.documentMode === 7))) {
```

$.browser was deprecatd in v1.3 and removed from starting with version 1.9 (http://api.jquery.com/jQuery.browser/). It is still available as a plugin though.

Are you able to replace the browser detection code with something more idiomatic to jQuery v1.9 or later (using  jQuery.support, see http://api.jquery.com/jQuery.browser/)?

**Comment by koenpunt** [comment]
Try the option_adding branch instead :)

**Comment by koenpunt** [comment]
Oh wait, same problem there

**Comment by ryanlchan** [comment]
Issue was fixed in harvesthq/chosen#950. What is the policy for determining when to pull in changes from harvesthq's master?

Happy to submit an individual PR for this change, though.

**Comment by koenpunt** [comment]
I've just updated my branch with the latest master, and also future updates are going to be much easier because I've rebased my branch completely.
